openedx/README.rst

```diff
@@ -0,0 +1,8 @@
+OpenEdx
+-------
+
+This is the root package for Open edX. 
+The intent is that all importable code from Open edX will eventually live here, including the code in the lms, cms, and common directories.
```

Comment:
In general please line-wrap documentation files to no more than 80 characters per line. This helps us more easily read Github diffs and comment on individual lines.

Since Ned is going to extend this documentation in the near future, I won't make you do this right now :)